docs/docs/05_flask_smorest/02_data_model_improvements/README.md

```diff
@@ -255,6 +257,8 @@ def create_store():
     return store
 ```
 
+Here we add a new import, [the `uuid` module](https://docs.python.org/3/library/uuid.html). We will be using it to create unique IDs for our stores and items instead of relying on the uniqueness of their `names`.
```

Comment:
```suggestion
Here we add a new import, [the `uuid` module](https://docs.python.org/3/library/uuid.html). We will be using it to create unique IDs for our stores and items instead of relying on the uniqueness of their names.
```

Since `names` isn't a variable in the code, I think better to not have it formatted as such 🤔